core/jazz_ui/src/app/pages/environment-logs/env-logs-section.component.scss

```diff
@@ -31,15 +31,69 @@
     display: block;
     margin-left: auto; // float: right;
     height: 32px;
-    width: 17rem;
-    .dropdown-btn {
+    /deep/.dropdown {
+      min-width: 11rem;
+    }
+    /deep/.dropdown-btn {
       height: 32px;
     }
+    /deep/ .dropdown-menu>li>a {
+      padding: 3px 12px;
+    }
   }
 }
 
-.filter-icon {
-  order: 1;
+
+.filter-wrap {
+  display: none;
+  position: absolute;
+  top: 0;
+  background-color: #ddd;
+  width: 250px;
+  right: 0;
+  padding: 1rem;
+  font-size: 1rem;
+  color: #000;
+  border-radius: 5px;
+  z-index: $zindex-dropdown;
+  transition: all 0.3s ease-in-out;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.15);
+  .filter-inner-wrap {
+    padding: 1rem;
+  }
+  .icon-icon-filter-unselected {
+    position: absolute;
+    right: 0;
+    top: 0;
+  }
+  .eachFilter {
+    border-bottom: 1px solid #aaa;
+    padding-bottom: 2rem;
+    font-weight: 900;
+    div {
+      line-height: 3.5rem;
+    }
+    &:last-child {
+      border-bottom: none;
+      padding: 0;
+    }
+  } // .eachFilter{
+  //     border-bottom: 1px solid #aaa;
```

Comment:
Remove the commented section if not needed

Reply:
fixed